### DISCUSSION TITLE
Error message in workflow to capture video

### DISCUSSION THREAD

**Discussion by JO-PANDA** [bonsai-rx/bonsai#2072] [Q&A]
![Skærmbillede 2024-12-04 122842](https://github.com/user-attachments/assets/59f212de-548b-4a4a-ad1e-3a1004e98a07)

Hello everyone, I am trying to get my Basler camera synchronized with the fiber photometry recordings. However, it is seemingly not working and I cannot find the mistake for it. what does the "no method overload arguments" error mean? I tried saving the video writer file with .avi behind and also change the four CC but cannot get it to work. 

If anyone has experience with setting especially Basler camera up to Bonsai via Python, I would be very appreciative in speaking with you.

**Comment by bruno-f-cruz** [comment]
This should help: https://github.com/orgs/bonsai-rx/discussions/1747#discussioncomment-9342806

**Reply by JO-PANDA** [reply]
Hi Bruno, Thank you for the reply, I am unsure of why the image transformer is needed when it is not shown in the bonsai manual?
